src/watchkit.cs

```diff
@@ -983,6 +1003,7 @@ interface WKExtension {
 
 		[Watch (4,0)]
 		[Export ("enableWaterLock")]
+		[Deprecated (PlatformName.WatchOS, 6,1, message: "Use WKInterfaceDevice.enableWaterLock")]
```

Comment:
Use C# (not ObjC) syntax + our format, e.g. `Use 'X' instead.` so

`"Use 'WKInterfaceDevice.EnableWaterLock ()' instead.`